mesheryctl/internal/cli/root/system/token.go

```diff
@@ -0,0 +1,170 @@
+package system
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var ctx string
+var tokenCmd = &cobra.Command{
+	Use:   "token",
+	Short: "Manage Meshery user tokens",
+	Long: `
+	Manipulate user tokens and their context assignments in your meshconfig`,
+	Args: cobra.MinimumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if ok := utils.IsValidSubcommand(availableSubcommands, args[0]); !ok {
+			return errors.New(utils.SystemError(fmt.Sprintf("invalid command: \"%s\"", args[0])))
+		}
+		return nil
+	},
+}
+
+var addTokenCmd = &cobra.Command{
+	Use:   "add",
+	Short: "Add a token to meshery config",
+	Example: `
+	mesheryctl system token add <token-name> -f <token-path>
+	mesheryctl system token add <token-name> (default path is auth.json)
+	`,
+	Args: cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		tokenName := args[0]
+		if tokenPath == "" {
+			tokenPath = "auth.json"
+		}
+
+		token := config.Token{
+			Name:     tokenName,
+			Location: tokenPath,
+		}
+		if err := config.AddTokenToConfig(token, utils.DefaultConfigPath); err != nil {
+			return errors.Wrap(err, "Could not add specified token to config")
+		}
+		log.Printf("Token %s added successfully!", tokenName)
+		return nil
+	},
+}
+var deleteTokenCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "Delete a token from meshery config",
+	Example: `
+	mesheryctl system token delete <token-name>
+	`,
+	Args: cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		tokenName := args[0]
+
+		token := config.Token{
+			Name:     tokenName,
+			Location: tokenPath,
+		}
+		if err := config.DeleteTokenFromConfig(token, utils.DefaultConfigPath); err != nil {
+			return errors.Wrap(err, "Could not delete specified token from config")
+		}
+		log.Printf("Token %s deleted successfully!", tokenName)
+		return nil
+	},
+}
+var setTokenCmd = &cobra.Command{
+	Use:   "set",
+	Short: "Set token for context",
+	Long:  "Set Token for current context or context passed with --context flag.",
```

Comment:
Why capital Token in long descr and lowercase in short descr. Let's go with lowercase.

Reply:
```suggestion
	Long:  "Set token for the current context or the context specified with the --context flag.",
```

Reply:
> @Revolyssup it would be great to support `mesheryctl system token view --all` - the `--all` flag. We'll want to provide a common set of subcommands and flags across all `system` resources (and non-system like `pattern` and `filter`).



> @Revolyssup it would be great to support `mesheryctl system token view --all` - the `--all` flag. We'll want to provide a common set of subcommands and flags across all `system` resources (and non-system like `pattern` and `filter`).

Yups, was actually adding it but it wasn't specified in issue. Added it now.